test/unit-tests/org/orgTest.js

```diff
@@ -267,6 +282,57 @@ describe('Test Org Controller', () => {
     })
   })
 
+  context('Getting an org\'s ID quota', () => {
+    it('Org with a negative ID quoto was not saved', (done) => {
+      let org = new Org(orgWithNegativeIdQuota);
+
+        org.validate((err) => {
+          expect(err.errors['policies.id_quota'].message).to.equal(CONSTANTS.MONGOOSE_VALIDATION.Org_policies_id_quota_min_message)
+          logger.info(CONSTANTS.MONGOOSE_VALIDATION.Org_policies_id_quota_min)
+          done()
+        });
+    });
+
+    it('Org has an available ID quoto of atleast zero', (done) => {
+      chai.request(server)
+        // perform the request to the api
+        .get('/api/test/cna/' + existentOrg.short_name + '/id_quota') 
+        .set(headers)
+        .end((err, res) => {
+          if (err) {
+            done(err)
+          }
+
+          // assert expected response
+          expect(res).to.have.status(200);
+          expect(res).to.have.property('body').and.to.be.a('object');
+          assert.isAtLeast(res.body.available, CONSTANTS.MONGOOSE_VALIDATION.Org_policies_id_quota_min, 'Org available ID quota is below the minimum threshold of ' + CONSTANTS.MONGOOSE_VALIDATION.Org_policies_id_quota_min + '.')
+          assert.isBelow(res.body.available, CONSTANTS.MONGOOSE_VALIDATION.Org_policies_id_quota_max, 'Org available ID quota exceeded maximum threshold of  ' + CONSTANTS.MONGOOSE_VALIDATION.Org_policies_id_quota_max + '.')
+          done()
+        });
+    });
+
+    it('Org\'s available ID quoto below zero returns an error', (done) => {
+      chai.request(server)
+        // perform the request to the api
+        .get('/api/test/cna/' + orgWithZeroIdQuota.short_name + '/id_quota') 
```

Comment:
`orgWithZeroIdQuota.short_name` should probably be replaced with `orgWithNegativeIdQuota.short_name`. To make this work, however, `orgWithNegativeIdQuota` must be added in the `before` and `after` hooks.